cmake/Install.cmake

```diff
@@ -540,11 +540,11 @@ endif ()
 
 ##########################################################   S Y S T E M    L I B R A R I E S   ######################################################
 
-# TODO: is this unecessary now? I had forgotten to actually create a Libraries via cpack_add_component but everything seemed fined
-# At worse, try not to uncomment this as is, but place it inside an if(PLATFORM) statement
-#SET(CMAKE_INSTALL_UCRT_LIBRARIES TRUE)
-#INCLUDE(InstallRequiredSystemLibraries)
-#INSTALL(FILES ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS} DESTINATION "./" COMPONENT Libraries)
+# Set to TRUE to install the Windows Universal CRT libraries for app-local deployment (e.g. to Windows XP).
+# This is meaningful only with MSVC from Visual Studio 2015 or higher, which is our case
+SET(CMAKE_INSTALL_UCRT_LIBRARIES TRUE)
+INCLUDE(InstallRequiredSystemLibraries)
+INSTALL(FILES ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS} DESTINATION "./" COMPONENT Libraries)
```

Comment:
I'm optimistic this will get things back to normal.  @JasonGlazer please confirm if you are able to do so.